subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -115,3 +121,29 @@ def enabled = providers.systemProperty("some-property").forUseAtConfigurationTim
 Support for using configuration caching with certain Gradle features is not yet implemented.
 
 - Composite builds
+
+
+[[testkit]]
+== Testing Build Logic with TestKit
+
+The Gradle TestKit (a.k.a. just TestKit) is a library that aids in testing Gradle plugins and build logic generally.
+For general guidance on how to use TestKit see the <<test_kit.adoc#test_kit,dedicated chapter>>.
+
+To enable configuration caching in your tests, you can pass the `--configuration-cache=on` argument to link:{javadocPath}/org/gradle/testkit/runner/GradleRunner.html[GradleRunner] or use one of the other methods described in <<configuration_cache.adoc#enable,Enabling the configuration cache>>.
```

Comment:
❓ do we have `--configuration-cache` as an alias for `--configuration-cache=on`? if so I would use the former.

Reply:
We don't yet, see https://github.com/gradle/gradle/issues/13170